docs/entities/rosetta/rosetta-construction-options.schema.json

```diff
@@ -52,6 +52,10 @@
     "fee": {
       "type": "string",
       "description": "Fee for this transaction"
+    },
+    "size": {
+      "type": "number",
```

Comment:
Should this be `integer`?

Reply:
Yep. Probably.

Reply:
Does it actually change index.d.ts ? I changed locally but I have the same file generated.

Reply:
It won't change the resulting type (TypeScript doesn't differentiate numbers and "integers"), but it would be more accurate were to use these specs for tests. `1.23` would fail with `integer` type